Quadriga/src/main/webapp/WEB-INF/views/navigation/nav-settings.jsp

```diff
@@ -12,19 +12,23 @@
 <div>
     <ul>
         <li><a
-            href="${pageContext.servletContext.contextPath}/auth/workbench/projects/${project.projectId}"><span
+            href="${pageContext.servletContext.contextPath}/auth/workbench/projects/${publicpageprojectid}"><span
```

Comment:
yes, that's fine. eventually we'll probably write an aspect for that, but for now we have to do it by hand.